src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ImporterService.cs

```diff
@@ -100,6 +109,37 @@ public class ImporterService : IImporterService
             GeographicLevel.PlanningArea
         };
 
+        private static readonly List<GeographicLevel> AllowedSoloGeographicLevels = new List<GeographicLevel>
+        {
+            GeographicLevel.Provider,
+            GeographicLevel.School,
+        };
+
+        public static bool HasSoloAllowedGeographicLevel(HashSet<GeographicLevel> subjectGeographicLevels)
```

Comment:
I'm not sure that solo is the correct term since a file can have two levels, school **and** provider and still get imported.

Reply:
It only allows the importing to school or provider data is there is one Geographic level: a file with school and provider data would have no data imported. I created a test subject for this instance.